pkg/server/app/handlers_test.go

```diff
@@ -160,6 +160,29 @@ func TestInfoPermissionDenied(t *testing.T) {
 	}
 }
 
+func TestListSuccess(t *testing.T) {
+	fake := NewFakeOperations()
+	name := "teresa"
+	fake.(*FakeOperations).Storage[name] = &App{Name: name}
+	s := NewService(fake)
+	user := &storage.User{Email: "gopher@luizalabs.com"}
+	ctx := context.WithValue(context.Background(), "user", user)
+
+	if _, err := s.List(ctx, &appb.Empty{}); err != nil {
+		t.Error("Got error on list: ", err)
+	}
+}
+
+func TestListAppNotFound(t *testing.T) {
+	s := NewService(NewFakeOperations())
+	user := &storage.User{Email: "gopher@luizalabs.com"}
+	ctx := context.WithValue(context.Background(), "user", user)
+
+	if _, err := s.List(ctx, &appb.Empty{}); teresa_errors.Get(err) != nil {
+		t.Errorf("expected ErrNotFound, got %v", err)
+	}
```

Comment:
I was thinking about your question @drgarcia1986 and I have one doubt. Do I really need do this test? Make sense to test if a list really exist?